src/main/java/org/jboss/xavier/analytics/pojo/input/workload/inventory/VMWorkloadInventoryModel.java

```diff
@@ -36,6 +36,7 @@
     private String version;
     private String host_name;
     private boolean cpuAffinityNotNull;
+    private boolean hasUEFIBoot;
```

Comment:
Why it's a boolean if the spreadsheet says it will have values "EFI" or "BIOS" ? ( refer to spreadsheet  MA -OCP validation rules )

Reply:
And considering that the expected value is a string and it's the firmware type, do you think renaming this attribute to "firmware" would be better ?